recipes-bsp/u-boot/files/0026-board-siemens-iot2050-use-the-named-gpio-to-control-.patch

```diff
@@ -0,0 +1,30 @@
+From 5c9591c09d6ce2cb33f0967bc2252a2dbe6fd25f Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Wed, 12 Jan 2022 10:34:18 +0800
+Subject: [PATCH 26/28] board:siemens:iot2050: use the named gpio to control
+ the user-button
+
+Has already add the gpio controll base as the prefix, using the named gpio
+to control the user button
```

Comment:
Sentence does not parse for me yet. But it should be about something else anyway: WHY do we want this patch, what is it's benefit?

Reply:
Use 25 to control the button because it just falls in the mcu domain and happens to be able to control the gpio. The prefix is to indicate which domain is the 25th

Reply:
> Please update the commit log accordingly.

rewrite the commit message